ext/date/date_core.c

```diff
@@ -5524,16 +5536,18 @@ d_lite_sec(VALUE self)
 
 /*
  * call-seq:
- *    d.sec_fraction     ->  rational
- *    d.second_fraction  ->  rational
+ *   sec_fraction -> rational
  *
- * Returns the fractional part of the second.
+ * Returns the fractional part of the second in range
+ * (Rational(0, 1)..Rational(59, 1)):
```

Comment:
I think it can only be in the range `Rational(0, 1)...Rational(1, 1)` (it can't include `Rational(1/1)`).

Reply:
@BurdetteLamar did you miss this one?

Reply:
Duh.  Totally misread your comment (and my own text).  Will fix tomorrow.

Reply:
Fixed.